src/pkcs15init/pkcs15-iasecc.c

```diff
@@ -1749,7 +1749,7 @@ iasecc_store_data_object(struct sc_pkcs15_card *p15card, struct sc_profile *prof
 	}
```

Comment:
I think on the line above, you will have to set `file = NULL` to avoid double-free

Reply:
Fixed in e38d38a